src/StateService/Verification/VerificationContext.cs

```diff
@@ -0,0 +1,139 @@
+using Akka.Actor;
+using Neo.Cryptography;
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Network.P2P;
+using Neo.Network.P2P.Payloads;
+using Neo.Persistence;
+using Neo.Plugins.StateService.Storage;
+using Neo.SmartContract;
+using Neo.SmartContract.Native;
+using Neo.Wallets;
+using System;
+using System.Collections.Generic;
+
+namespace Neo.Plugins.StateService.Verification
+{
+    public class VerificationContext
+    {
+        private const uint MaxValidUntilBlockIncrement = 100;
```

Comment:
```suggestion
        private static const uint MaxValidUntilBlockIncrement = 100;
```

Reply:
Why?

Reply:
I misunderstood. `const` is enough. `static readonly` can be modified.